package.json

```diff
@@ -51,9 +51,9 @@
     "numeral": "2.0.6",
     "prando": "^3.0.3",
     "querystring": "^0.2.0",
-    "react": "16.3.2",
+    "react": "16.5.0",
     "react-i18next": "7.10.1",
-    "react-native": "0.55.4",
+    "react-native": "0.57-RC",
```

Comment:
how does this work? latest is `0.57.0-rc.4` , is it a semver thing? 🤔

Reply:
will fix but I do have the correct one installed